### ISSUE TITLE
Server-side SDKs don't provide access to trait data

### ISSUE THREAD

**Issue by dabeeeenster** [SolidStateGroup/bullet-train-api#2767]
If you Identify a user within the SDKS, you are not able to access the traits for that Identity. Looking at the python and .net code, this is the case at least. Most likely it is the same for the other SDKs. 

We should provide a way to access that data.

**Comment by matthewelwell** [comment]
I tend to disagree with this issue. The reason that we don't expose this data is because the Flagsmith application is not designed to be used as a user data storage platform. I feel like having this functionality would encourage people to abuse the Flagsmith application by storing and using traits in ways other than for user segmenting (their only purpose in Flagsmith).
